features/priority_failover.feature

```diff
@@ -0,0 +1,20 @@
+Feature: priority replication
+  We should check that we can give nodes priority during failover
+
+  Scenario: check failover priority 0 prevents leaderships
+    Given I configure and start postgres0 with a tag failover_priority 1
+    And I configure and start postgres1 with a tag failover_priority 0
+    When I shut down postgres0
```

Comment:
```suggestion
    Then replication works from postgres0 to postgres1 after 20 seconds
    When I shut down postgres0
```

Reply:
This helps, I also think that sometimes postgres3 would become leader so fast that postgres2 would see itself as having a inferior WAL status, which would lead to a different log message. Fixed the behave step to include accepting either of these two cases (either node 2 sees node 3 with higher priority _or_ higher wal status because it's already become the leader)

Reply:
Looks like it's still failing behave, but on a different test that seems unrelated